servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/MessageBodyTests.java

```diff
@@ -13,61 +13,61 @@
 public class MessageBodyTests {
 
     @Test
-    public void NullBinaryDataTest() {
+    public void nullBinaryDataTest() {
```

Comment:
Even though the class and method are public,  but it is a test class. It should not be a code-breaking change. Same to rest of methods in the class